### PR TITLE
[onert] Revisit generator map in OperationFactory

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit removes deprecated OPs generator map and remove code duplication.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: https://github.com/Samsung/ONE/issues/8786

### CODE REVIEW

File: runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc [hunk 1]
```diff
@@ -940,24 +830,7 @@ OperationFactory::OperationFactory()
...
-    //  2 -> Input Y(false) Tensor Index
-    OperandIndexSequence inputs{init_param.inputs[0], init_param.inputs[1], init_param.inputs[2]};
-
-    return new operation::Select{inputs, outputs};
-  };
-
-  // ANEURALNETWORKS_RSQRT_EX is deprecated
-  // TODO Remove ANEURALNETWORKS_RSQRT_EX
-  _map[ANEURALNETWORKS_RSQRT_EX] = _map[ANEURALNETWORKS_RSQRT];
+  _map[ANEURALNETWORKS_SELECT_V2_EX] = _map[ANEURALNETWORKS_SELECT];
```
Comment (hseok-oh) [inline runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc:833]: Generator is same.

**Review by jyoungyun — APPROVED** [review]
LGTM

File: tests/nnapi/specs/V1_2/equal_broadcast_float_nnfw.mod.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 i1 = Input("op1", "TENSOR_FLOAT32", "{2, 2}")
 i2 = Input("op2", "TENSOR_FLOAT32", "{1, 2}")
 
-i3 = Output("op3", "TENSOR_QUANT8_ASYMM", "{2, 2}, 1.f, 0")
-model = model.Operation("EQUAL_EX", i1, i2).To(i3)
```
Comment (ragmani) [inline tests/nnapi/specs/V1_2/equal_broadcast_float_nnfw.mod.py:7]: EQUAL_EX is going to be deprecated?

Comment (hseok-oh) [reply tests/nnapi/specs/V1_2/equal_broadcast_float_nnfw.mod.py:7]: Right

**Pull request merged**